Automattic/jetpack
# Handle cloud css timeout

Fixes 40-gh-Automattic/boost-shield-api

#### Changes proposed in this Pull Request:
* Mark critical CSS request as failed if the request is still pending after one hour, i.e. No response from server.

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Request cloud CSS
* Make sure the server would not respond for one hour
* Check and make sure that the request is marked as failed if you check status after 1 hour.
